sites/main-page-title/includes/main-page-title.html

```diff
@@ -0,0 +1,12 @@
+{% if include %}
+	{% if include.dir == 'ltr' or include.dir == 'rtl' %}
+		<h1 property="name"
+		{%- if include.rdfa -%}
+		typeof="WebPageElement"
```

Comment:
The h1 don't have a `typeof` attribute. That `WebPageElement` must be defined on a element that wrap the whole content, like the `<main>` element.